CredentialCard.vue

```diff
@@ -13,7 +13,7 @@
             <q-item-label
               class="text-left text-subtitle1"
               lines="2">
-              {{credential.name}}
+              {{credential.name || credential.type[1]}}
```

Comment:
I don't think we can do this, this information doesn't make sense to the user and could even be inaccurate. For now, we should just literally say "Credential" or "Untitled Credential". We're going to need to use other meta information, I think, such as the website where the user retrieved the credential, to display more.

Reply:
Addressed: d93d49952fdddf521f15dfcfeb79d08e2e3cb5d1